sdk/go/eks/cluster.go

```diff
@@ -13,7 +13,6 @@ import (
 	"github.com/pulumi/pulumi-aws/sdk/v6/go/aws/iam"
 	"github.com/pulumi/pulumi-eks/sdk/v2/go/eks/utilities"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
-	"github.com/pulumi/pulumi/sdk/v3/go/pulumix"
```

Comment:
is this dropped due to the pu/pu version bump?

Reply:
Yes, I believe it's pulumi/pulumi#14492.